WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/StatsListViewModel.kt

```diff
@@ -27,6 +27,7 @@ import org.wordpress.android.ui.stats.refresh.lists.StatsListViewModel.StatsSect
 import org.wordpress.android.ui.stats.refresh.utils.ItemPopupMenuHandler
 import org.wordpress.android.ui.stats.refresh.utils.NewsCardHandler
 import org.wordpress.android.ui.stats.refresh.utils.StatsDateSelector
+import org.wordpress.android.ui.stats.refresh.utils.StatsDateSelector.Factory
```

Comment:
I did some modifications in this class that was later rolled back. I kept this one because it solved an Android Studio warning. I am fine with both, please let me know what you think it's best.

Reply:
Fixed in [af23b7e](https://github.com/wordpress-mobile/WordPress-Android/pull/13047/commits/af23b7e98f96188ee4c047c25e92eb9f8d586c3c)